saleor/graphql/shop/utils.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
+
+from django_countries import countries
+
+from ...shipping.models import ShippingZone
+
+
+def get_countries_codes_list(in_shipping_zones: Optional[bool] = None):
+    """Return set of countries codes."""
```

Comment:
We should also describe the parameter (it could have the same description as in the schema).